src/headless-cms/components/cmsSidebarLayout.module.scss

```diff
@@ -0,0 +1,22 @@
+@use 'layout';
+
+.cmsSidebarLayout {
+  display: flex;
+  flex-direction: column;
+
+  @include layout.respond-above(md) {
+    flex-direction: row;
+  }
+
+  &Main {
+    @include layout.respond-above(md) {
+      flex-basis: 66.666%;
+    }
+  }
+
+  &Aside {
+    @include layout.respond-above(md) {
+      flex-basis: 33.333%;
+    }
+  }
+}
```

Comment:
Hmm. I'm surprised that you chose to use flexbox when styling this layout 🤔 Did you consider grid?

My solution would be 

```
.cmsSidebarLayout {
  display: grid;

  @include layout.respond-above(md) {
    grid-template-columns: 2fr 1fr;
  }
}

```

Reply:
I used flex because this is a one dimensional layout and in general the guideline tends to be to use flex with 1D and grid with 2D.